test/scripts/openshift-ci/deploy.kserve-manual.sh

```
@@ -0,0 +1,119 @@
+fi
+
+echo "KSERVE_CONTROLLER_IMAGE=$KSERVE_CONTROLLER_IMAGE"
+echo "LLMISVC_CONTROLLER_IMAGE=$LLMISVC_CONTROLLER_IMAGE"
+echo "KSERVE_AGENT_IMAGE=$KSERVE_AGENT_IMAGE"
+echo "KSERVE_ROUTER_IMAGE=$KSERVE_ROUTER_IMAGE"
+echo "STORAGE_INITIALIZER_IMAGE=$STORAGE_INITIALIZER_IMAGE"
+
+echo "Installing KServe via kustomize..."
+
+cp "${PARAMS_ENV}" "${PARAMS_ENV}.bak"
+trap "mv '${PARAMS_ENV}.bak' '${PARAMS_ENV}' 2>/dev/null || true" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)